bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -297,12 +299,29 @@ def construct_command(self):
         """
         raise NotImplementedError("Subclasses should implement this!")
 
+    def validate_output(self):
+
+        def _parent_dir(d):
+            return os.path.abspath(os.path.join(d, os.path.pardir))
+
+        abs_path_of_allowed_dirs = map(os.path.abspath, self.config.general_config['allowed_output_folders'])
```

Comment:
It's probably just paranoia, but how about requiring absolutes paths in the config? Or is it important for the usage to be able to supply relative paths? I'm mostly thinking in terms of the validation logic being as simple as possible. Having said that, this is pretty straightforward already.